osu.Game/Overlays/MusicController.cs

```diff
@@ -81,6 +81,11 @@ private void load()
             mods.BindValueChanged(_ => ResetTrackAdjustments(), true);
         }
 
+        /// <summary>
+        /// Forcefully reload the current <see cref="WorkingBeatmap"/>'s track from disk.
+        /// </summary>
+        public void ForceReloadCurrentBeatmap() => changeTrack();
```

Comment:
Would probably rename to `ForceReloadCurrentTrack`. Reloading beatmap in music controller sounds a little misleading.